meta-facebook/wc-mb/src/shell/commands/plat_info_shell.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "plat_info_shell.h"
+#include "plat_class.h"
+
+int cmd_wcmb_info_print(const struct shell *shell, size_t argc, char **argv)
```

Comment:
I think we should make the `cmd_info_print (...)` __weak and just override it to include extra information.
I don't think it's necessary to make a different class of commands just for wc.

Instead we can add them to platform commands if we need to add new commands in the future.